Assets/Scripts/ComboBehaviour.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ComboBehaviour : MonoBehaviour
+{
+    public float maxCombo;
+    public bool canInstantKill = false;
+
+
+    [HideInInspector] public float currentCombo = 0;
+    [SerializeField] float comboDecayRate;
+    private Coroutine coro_ComboDecay;
+    private bool isDecaying = false;
+    
+
+    private void Awake()
+    {
+        currentCombo = 0;
+        canInstantKill = false;
+    }
+    void Start()
+    {
+        isDecaying = true;
+        coro_ComboDecay = StartCoroutine(ComboDecay());
+    }   
+
+    public void Trigger_AddCombo(float _delta)
+    {
+        StartCoroutine(DelayAddCombo(_delta));
+    }
+
+    IEnumerator DelayAddCombo(float _delta)
+    {
+        /////DELAY ADD COMBO TO PREVENT GLITCHES
```

Comment:
too many slashes, i know you're trying to make a title but the caps alone suffice for that intention. just use two